### ISSUE TITLE
Improve the alignment of GenerationTool INFO log

### ISSUE THREAD

**Issue by lukaseder** [jOOQ/jOOQ#11424]
The `GenerationTool` logs the configuration it has loaded, including all the forced flag values, to the `INFO` log. It is currently not well aligned:

```
[INFO] JavaGenerator parameters
[INFO] ----------------------------------------------------------
[INFO]   annotations (generated): false
[INFO]   annotations (JPA: any) : false
[INFO]   annotations (JPA: version):
[INFO]   annotations (validation): false
[INFO]   comments               : true
[INFO]   comments on attributes : true
[INFO]   comments on catalogs   : true
[INFO]   comments on columns    : true
[INFO]   comments on keys       : true
[INFO]   comments on links      : true
[INFO]   comments on packages   : true
[INFO]   comments on parameters : true
[INFO]   comments on queues     : true
[INFO]   comments on routines   : true
[INFO]   comments on schemas    : true
[INFO]   comments on sequences  : true
[INFO]   comments on tables     : true
[INFO]   comments on udts       : true
[INFO]   sources                : true
[INFO]   sources on views       : true
[INFO]   daos                   : false
[INFO]   deprecated code        : false
[INFO]   global references (any): true
[INFO]   global references (catalogs): true
[INFO]   global references (keys): true
[INFO]   global references (links): true
[INFO]   global references (queues): true
[INFO]   global references (routines): true
[INFO]   global references (schemas): true
[INFO]   global references (sequences): true
[INFO]   global references (tables): true
[INFO]   global references (udts): true
[INFO]   indexes                : true
[INFO]   instance fields        : true
[INFO]   interfaces             : false
[INFO]   interfaces (immutable) : false
[INFO]   javadoc                : true
[INFO]   keys                   : true
[INFO]   links                  : true
[INFO]   pojos                  : false
[INFO]   pojos (immutable)      : false
[INFO]   queues                 : true
[INFO]   records                : true
[INFO]   routines               : true
[INFO]   sequences              : true
[INFO]   sequenceFlags          : true
[INFO]   table-valued functions : true
[INFO]   tables                 : true
[INFO]   udts                   : true
[INFO]   relations              : true
[INFO] ----------------------------------------------------------
```

**Comment by lukaseder** [comment]
Fixing this, now. The best approach is to generate some "header" logs (e.g. `comments`) and then don't repeat that word for each sub-flag.

I'll also add a few missing items to the list.

**Comment by lukaseder** [comment]
The output now looks like this:

```
11:57:49,644 [INFO] JavaGenerator parameters
11:57:49,645 [INFO] ----------------------------------------------------------
11:57:49,648 [INFO] annotations
11:57:49,652 [INFO]   generated              : false
11:57:49,654 [INFO]   JPA                    : false
11:57:49,656 [INFO]   JPA version            :
11:57:49,658 [INFO]   validation             : false
11:57:49,660 [INFO]   spring                 : false
11:57:49,662 [INFO] comments
11:57:49,663 [INFO]   comments               : true
11:57:49,665 [INFO]   on attributes          : true
11:57:49,667 [INFO]   on catalogs            : true
11:57:49,669 [INFO]   on columns             : true
11:57:49,671 [INFO]   on embeddables         : true
11:57:49,673 [INFO]   on keys                : true
11:57:49,675 [INFO]   on links               : true
11:57:49,677 [INFO]   on packages            : true
11:57:49,679 [INFO]   on parameters          : true
11:57:49,680 [INFO]   on queues              : true
11:57:49,682 [INFO]   on routines            : true
11:57:49,684 [INFO]   on schemas             : true
11:57:49,687 [INFO]   on sequences           : true
11:57:49,689 [INFO]   on tables              : true
11:57:49,690 [INFO]   on udts                : true
11:57:49,692 [INFO] sources
11:57:49,694 [INFO]   sources                : true
11:57:49,695 [INFO]   sources on views       : true
11:57:49,697 [INFO] global references
11:57:49,699 [INFO]   global references      : true
11:57:49,701 [INFO]   catalogs               : true
11:57:49,702 [INFO]   domains                : true
11:57:49,704 [INFO]   indexes                : true
11:57:49,706 [INFO]   keys                   : true
11:57:49,708 [INFO]   links                  : true
11:57:49,709 [INFO]   queues                 : true
11:57:49,711 [INFO]   routines               : true
11:57:49,713 [INFO]   schemas                : true
11:57:49,715 [INFO]   sequences              : true
11:57:49,716 [INFO]   tables                 : true
11:57:49,718 [INFO]   udts                   : true
11:57:49,720 [INFO] object types
11:57:49,721 [INFO]   daos                   : false
11:57:49,723 [INFO]   indexes                : true
11:57:49,725 [INFO]   instance fields        : true
11:57:49,727 [INFO]   interfaces             : false
11:57:49,729 [INFO]   interfaces (immutable) : false
11:57:49,730 [INFO]   javadoc                : true
11:57:49,732 [INFO]   keys                   : true
11:57:49,734 [INFO]   links                  : true
11:57:49,736 [INFO]   pojos                  : false
11:57:49,738 [INFO]   pojos (immutable)      : false
11:57:49,740 [INFO]   queues                 : true
11:57:49,741 [INFO]   records                : true
11:57:49,743 [INFO]   routines               : true
11:57:49,745 [INFO]   sequences              : true
11:57:49,747 [INFO]   sequenceFlags          : true
11:57:49,748 [INFO]   table-valued functions : true
11:57:49,750 [INFO]   tables                 : true
11:57:49,752 [INFO]   udts                   : true
11:57:49,754 [INFO]   relations              : true
11:57:49,755 [INFO] other
11:57:49,757 [INFO]   deprecated code        : false
11:57:49,759 [INFO] ----------------------------------------------------------
```

**Comment by lukaseder** [comment]
Fixed in jOOQ 3.17.0
